db/fixtures/ae_datastore/ManageIQ/System/CommonMethods/Quota.class/__methods__/validate_quota.rb

```diff
@@ -0,0 +1,457 @@
+
+def request_info
+  type = $evm.root['quota_type']

```

Comment:
@tinaafitz 
Is the quota_type going to be specified in the instance or in the root object?